### PR TITLE
format files and add format script

### PR DESCRIPTION
**Pull request by avneesh0612**
The files weren't formatted and there was no script for formatting them so added the prettier one. We can also add this in pre commit

### CODE REVIEW

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
can you use this prettier config?

{
  "singleQuote": false,
  "trailingComma": "all",
  "importOrderSeparation": true,
  "importOrderSortSpecifiers": true
}

**Comment by avneesh0612** [comment]
Adding 
```"
"importOrderSeparation": true,
"importOrderSortSpecifiers": true
```

would just give this warning- [warn] Ignored unknown option { importOrderSeparation: true, importOrderSortSpecifiers: true }. And makes no difference

File: portal/guides/sell-nft-multiple-currencies.mdx [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
```
Comment (nachoiacovino) [inline portal/guides/sell-nft-multiple-currencies.mdx]: why is this in this branch? :D

**Review by nachoiacovino — CHANGES_REQUESTED** [review]

File: portal/guides/sell-nft-multiple-currencies.mdx (hunk 1 above)
Comment (avneesh0612) [reply portal/guides/sell-nft-multiple-currencies.mdx]: Oh rip lmao

**Pull request closed without merging**